src/Patches/GhostHints.cs

```diff
@@ -334,6 +367,13 @@ public class HintGhost {
                     if (PaletteEditor.CelShadingEnabled && PaletteEditor.ToonFox != null) {
                         NewGhostFox.transform.GetChild(2).GetChild(1).GetComponent<SkinnedMeshRenderer>().material = PaletteEditor.ToonFox.GetComponent<MeshRenderer>().material;
                     }
+                    
+                    if (HintGhost.Fishing == true) {
+                        GameObject fishingRod = GameObject.Instantiate(ModelSwaps.FishingRod);
+                        fishingRod.SetActive(true);
+                        fishingRod.transform.position = HintGhost.FishingRodPos.pos;
+                        fishingRod.transform.rotation = HintGhost.FishingRodPos.rot;
+                    }
```

Comment:
```suggestion
```suggestion
                    if (HintGhost.Fishing) {
                        GameObject fishingRod = GameObject.Instantiate(ModelSwaps.FishingRod, HintGhost.FishingRodPos.pos, HintGhost.FishingRodPos.rot);
                        fishingRod.SetActive(true);
                    }
```